config/manager/kustomization.yaml

```diff
@@ -4,5 +4,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: ghcr.io/k8sgpt-ai/k8sgpt-operator
-  newTag: feature-integration-support-4
+  newName: tibbar/k8sgpt-operator
```

Comment:
is this for testing in local env?

Reply:
yeah, this isn't important. It gets overwritten